test/e2e/framework/util.go

```diff
@@ -552,6 +552,14 @@ func LoadConfig() (config *restclient.Config, err error) {
 	}
 	c, err := restclientConfig(TestContext.KubeContext)
 	if err != nil {
+		// In case Host is not set in TestContext, sets it as
+		// CurrentContext Server for k8s API client to connect to.
+		if TestContext.Host == "" && c != nil && c.Clusters != nil {
```

Comment:
as i pointed out earlier if `err != nil`, `c` will be always `nil`.

Reply:
My bad. I misread this part. Fixed now.